Source/embeddings/index.ts

```diff
@@ -1,20 +1,24 @@
 // Copyright (c) Dolittle. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+import { CouldNotResolveDeletionToEvents } from '@dolittle/sdk.embeddings/CouldNotResolveDeletionToEvents';
+import { CouldNotResolveUpdateToEvents } from '@dolittle/sdk.embeddings/CouldNotResolveUpdateToEvents';
```

Comment:
```suggestion
```
These imports shouldn't be here

Reply:
Removed, and you are spot on. They shouldn't be here. The shame is real :).